### DISCUSSION TITLE
Motion detectors set to "wall" detect submarine bounding boxes

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15399] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

they detect the gigantic bounding box around submarines, not the walls themselves. 

![image](https://github.com/user-attachments/assets/e3811f45-200d-4bdf-900e-44a9eff68df6)
(a picture of a motion detector going off when the submarine wall is nowhere near the detection zone)

![image](https://github.com/user-attachments/assets/179a3331-b998-4275-9756-16c38c17a58f)
these massive areas should not be detected as a wall. what if there was a shuttle with a motion detector docked there? it would always activate before getting anywhere near the submarine, which is not the intent. it should detect the walls' hitboxes.

![image](https://github.com/user-attachments/assets/14a740e6-24fe-428f-84ab-277c304e855c)
it gets worse and worse when you get into custom maps or submarines etc

this submarine is immune to nuclear torpedos! why? because they detonate before getting close lol
![image](https://github.com/user-attachments/assets/f3cdd9b4-f880-457f-93a2-eaacdf1b15fa)

### Reproduction steps

1. try to use motion detector to detect wall
2. lol!!!!!! it detects bounding boxes instead! trolled!

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this and motion detectors are only activated by the wall themself
Closing discussion as resolved

**Reply by girlyguppy** [reply]
Me when I accidentally find a gigantic bug and report it and it gets fixed 😀
